.github/workflows/create-docker-images.yml

```
@@ -0,0 +1,30 @@
+# Creates a new docker image and pushes it to DockerHub when a version tag is created.
+name: create-docker-images
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  # Based on instructions from https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
+  build-frontend-and-backend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

We can use the most recent version here:
```suggestion
        uses: actions/checkout@v4
```